cardano-db/src/Cardano/Db/Schema.hs

```diff
@@ -113,8 +113,7 @@ share
 
     -- New for Alonzo
     validContract       Bool                                    -- False if the contract is invalid, True otherwise.
-    exUnitFee           DbLovelace          sqltype=lovelace
-    scriptSize          Word64              sqltype=uinteger
+    scriptSize          DbWord64            sqltype=word64type
```

Comment:
I actually think `Word64` would be fine there.

`DbWor64` maps to a PostgreSQL `numeric` type and should only be required for things where all possible values are likely. As for `scriptSize` they simply are not allowed to be that big (worst case the size of a transaction would be too big before there is any chance of that overflowing.

`unitneger` is probably OK, but if you really wanted something mostly guaranteed to to be right, try `Word64` with `sqltype=word63type` which maps for PostgreSQL `bigint` (64 bit integer) with a constraint that it be `>= 0`.